### PR TITLE
Modified outdated section in Step 6 (fixes #2119)

### LINKED ISSUE #2119: Outdated section in Step 6
### Problem
A particular section in [Step 6](http://open-learning-exchange.github.io/#!./pages/vi/vi-github-issues.md) detailing the instructions to submit a new issue appears to be outdated.  The 'proposed solution' section is now included with the original issue instead of in an additional comment, so the instructions may be misleading.

### Screenshots
    
issue2  
issue

### Proposed solution

Rewrite the paragraph above the screenshot in Step 6 so that it includes the necessary information, and use a newer issue as an example.

**For example:**  
First, you will need to open an issue within the right repository ([upstream repository](https://github.com/open-learning-exchange/open-learning-exchange.github.io)) and explain the problem. If you are explaining a bug or how to replicate an issue, please be as explicit as possible and use pictures as necessary.  You will then research the possible solutions or ideas on how to address the Issue before detailing your proposed solution, as you can see here (<- link here). Someone else will hopefully provide input (as Dogi did here), and then you can move ahead with making the proposed changes.

### PR DESCRIPTION
**Pull request by jazdao**
Fixes #2119

### Description
A particular section in [Step 6](http://open-learning-exchange.github.io/#!./pages/vi/vi-github-issues.md) detailing the instructions to submit a new issue is outdated.  The 'proposed solution' section is now included with the original issue instead of in an additional comment, so the instructions may be misleading.  This is resolved by rewriting the paragraph above the screenshot in Step 6 so that it includes the necessary information and also uses a relevant example.

### RawGit preview link

https://raw.githack.com/jazdao/jazdao.github.io/outdated-section/#!pages/vi/vi-github-issues.md

### CODE REVIEW

**Comment by inDepthh** [comment]
@jazdao Looks like you still need to update the example image.

**Comment by jazdao** [comment]
Hi @inDepthh, I don't think I can get a better screenshot than the current one.  On mine, it looks like this:
screenshot

**Comment by inDepthh** [comment]
You can extend the box by dragging the 3 lines located directly under the scrollbar

**Comment by jazdao** [comment]
@inDepthh  Okay, thanks!

**Comment by inDepthh** [comment]
@jazdao On second thought, you could use a real issue example instead of the template if you want

**Comment by jazdao** [comment]
@inDepthh Okay, I'll make the changes.  In that case, is it necessary to link to an example?

**Comment by inDepthh** [comment]
@jazdao Nevermind it will work. My bad, I'm trying to fix what is not broken,

File: pages/vi/vi-github-issues.md [hunk 1]
```diff
@@ -20,11 +20,9 @@ Below, you'll find the workflow where we brought up a problem, researched an ans
...
 
-![Providing Detail on New Issue](images/vi-new-issue-format.png)
-
-After you have submitted the new Issue, you will then research the possible solutions, or ideas, on how to address the Issue. Once you've completed your research, then you will add a comment detailing your proposed solution, as you can see [here](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/15). Someone else will hopefully provide input (as Dogi did here) and then you can move ahead with making the proposed changes.
+![Providing Detail on New Issue](images/vi-new-issue-format.JPG)
```
Comment (lmmrssa) [inline pages/vi/vi-github-issues.md]: @jazdao approving it but try to keep filename and extension on lowercase.

**Pull request merged**